flutter/flutter_todo/android/app/src/main/java/com/example/flutter_todo/FRAuthSampleBridge.java

```diff
@@ -0,0 +1,607 @@
+/*
```

Comment:
in future , please change this to kotlin, because we are going to use lot of suspend methods. that would be tricky to call from java

Reply:
yeah when we re-write that we should do Kotlin. not a priority at the moment though